custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql

```diff
@@ -0,0 +1,113 @@
+          ELSE 4 END AS current_month_nutrition_status_sort,
+      CASE WHEN child_health_monthly.current_month_stunting = 'severe' THEN 1
+          WHEN child_health_monthly.current_month_stunting = 'moderate' THEN 2
+          WHEN child_health_monthly.current_month_stunting = 'normal' THEN 3
+          ELSE 4 END AS current_month_stunting_sort,
+      CASE WHEN child_health_monthly.current_month_wasting = 'severe' THEN 1
+          WHEN child_health_monthly.current_month_wasting = 'moderate' THEN 2
+          WHEN child_health_monthly.current_month_wasting = 'normal' THEN 3
+          ELSE 4 END AS current_month_wasting_sort,
+      CASE
+        WHEN child_health_monthly.zscore_grading_hfa = 1 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN 'severe'
+        WHEN child_health_monthly.zscore_grading_hfa = 2 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN 'moderate'
+        WHEN child_health_monthly.zscore_grading_hfa = 3 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN 'normal'
+        ELSE '' END AS current_month_stunting_v2,
+      CASE
+        WHEN (child_health_monthly.zscore_grading_wfh = 1 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 1 AND child_health_monthly.muac_grading_recorded_in_month = 1) THEN 'severe'
+        WHEN (child_health_monthly.zscore_grading_wfh = 2 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 2 AND child_health_monthly.muac_grading_recorded_in_month = 2) THEN 'moderate'
+        WHEN (child_health_monthly.zscore_grading_wfh = 3 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 3 AND child_health_monthly.muac_grading_recorded_in_month = 3) THEN 'normal'
+        ELSE '' END AS current_month_wasting_v2,
+      CASE
+        WHEN child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN child_health_monthly.zscore_grading_hfa
+        ELSE 4 END AS current_month_stunting_v2_sort,
+      CASE
+        WHEN (child_health_monthly.zscore_grading_wfh = 1 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 1 AND child_health_monthly.muac_grading_recorded_in_month = 1) THEN 1
+        WHEN (child_health_monthly.zscore_grading_wfh = 2 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 2 AND child_health_monthly.muac_grading_recorded_in_month = 2) THEN 2
+        WHEN (child_health_monthly.zscore_grading_wfh = 3 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 3 AND child_health_monthly.muac_grading_recorded_in_month = 3) THEN 3
+        ELSE 4 END AS current_month_wasting_v2_sort
+  FROM "public"."awc_location_months" "awc_location_months"
```

Comment:
It looks like you moved the view files and change the definition in one commit. Could you split those up to two different commits?

Reply:
@emord I split to two commits, in the first I moved other views definition to separate files, and in the second I moved ```child_health_monthly_view``` definition to the same directory and I added changes.